sklearn/tests/test_common.py

```diff
@@ -83,16 +85,20 @@ def test_all_estimators():
 
 def test_non_meta_estimators():
     # input validation etc for non-meta estimators
-    # FIXME these should be done also for non-mixin estimators!
-    estimators = all_estimators(type_filter=['classifier', 'regressor',
-                                             'transformer', 'cluster'])
+    estimators = all_estimators()
```

Comment:
Just to emphasize, this is the important line in this PR. It extends testing to many more estimators, namely those that don't inherit from one of the four mixins used before.